scripts/dataset_processing/tts/aishell3/get_data.py

```diff
@@ -102,18 +105,26 @@ def __process_transcript(file_path: str):
             processed_file = file_path / "processed" / wav_name
             # convert wav to mono 22050HZ, 16 bit (as SFSpeech dataset)
             subprocess.run(f"sox {wav_file} -r 22050 -c 1 -b 16 {processed_file}", shell=True)
-            simplified_text = cc.convert(text)
-            normalized_text = normalizer_call(simplified_text)
-            entry = {
-                'audio_filepath': os.path.abspath(processed_file),
-                'duration': float(duration),
-                'text': text,
-                'normalized_text': normalized_text,
-                'speaker': int(speaker[3:]),
-            }
-
-            i += 1
-            entries.append(entry)
+            candidates.append((processed_file, duration, text, speaker))
+
+    # remapping the speakder to speaker_id (start from 1)
+    remapping = {}
+    for index, speaker in enumerate(sorted(speakers)):
+        remapping[speaker] = index + 1
+
+    for processed_file, duration, text, speaker in candidates:
+        simplified_text = cc.convert(text)
+        normalized_text = normalizer_call(simplified_text)
+        entry = {
```

Comment:
it would be great if adding a new key/val, e.g. `"speaker_raw": speaker`, to store the mapping in the JSON manifest since there is nowhere storing the remap file.

Reply:
> it would be great if adding a new key/val, e.g. `"speaker_raw": speaker`, to store the mapping in the JSON manifest since there is nowhere storing the remap file.

Thanks for the suggestion.
Done.